components/profile/ProfileDetail.vue

```diff
@@ -144,13 +146,25 @@
           </div>
         </div>
         <hr class="my-0" />
-        <ItemsGrid :search="itemsGridSearch" />
+        <template v-for="tab in itemTabs">
+          <ItemsGrid
+            v-show="activeTab === tab"
+            :key="tab"
+            :search="getItemGridSearch(tab)"
+            @total="(count) => handleTabTotal(count, tab)" />
+        </template>
```

Comment:
potential performance hit

I don't think it makes sense to load multiple item grids at once
it will work okay while there are a small number of images  / total items
but for a large number, the user will wait while images on tab he isn't looking at are loading


I think you will need a minimal query in order to know how many items are in each tab

Reply:
yeah I wasn't really sure since this way it end up making the call for each tab even if you didn't click on them , performance hit there , so as you pointed out i'll make a small query to fetch the count